back-end/src/sciler/handler/handler.go

```diff
@@ -144,7 +192,7 @@ func (handler *Handler) onInstructionMsg(raw Message) {
 	logrus.Info("instruction message received from: ", raw.DeviceID)
 	if raw.Contents["instruction"] == "test all" && raw.DeviceID == "front-end" { // TODO maybe switch again
 		message := Message{
-			DeviceID: "back-end",
```

Comment:
This should be back-end, this makes the logs unreadable.
TODO: make additional field: confirmation-sender, to which a confirmation is sent if not empty

Reply:
made issue GH-80